### ISSUE TITLE
Clean up build logs

### ISSUE THREAD

**Issue by pnodet** [NotionX/react-notion-x#524]
Hi! 👋 
      
Firstly, thanks for your work on this project! 🙂

Today I used [patch-package](https://github.com/ds300/patch-package) to patch `notion-compat@6.16.0` for the project I'm working on.

We use nextjs to build a static site using your packages and it works great but on build the logs are filled with blocks info that are not relevant for me, and maybe for many apart from you on debugs. Would you consider removing this?

Thanks!

Here is the diff that solved my problem:

```diff
diff --git a/node_modules/notion-compat/build/index.js b/node_modules/notion-compat/build/index.js
index 4d51e77..c23ecbc 100644
--- a/node_modules/notion-compat/build/index.js
+++ b/node_modules/notion-compat/build/index.js
@@ -692,7 +692,6 @@ var NotionCompatAPI = class {
     let blocks = [];
     let cursor;
     do {
-      console.log("blocks.children.list", { blockId, cursor });
       const res = await this.client.blocks.children.list({
         block_id: blockId,
         start_cursor: cursor
```

<em>This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296).</em>

**Comment by pnodet** [comment]
Sorry, all good this was fixed in [#499](https://github.com/NotionX/react-notion-x/pull/499)

**Comment by pnodet** [comment]
This issue should be fixed in v6.16.1 but npm is still at 6.16.0

**Comment by transitive-bullshit** [comment]
Released latest v7
